ddl/index.go

```diff
@@ -1412,14 +1389,13 @@ func loadDDLReorgVars(w *worker) error {
 //	4. Wait all these running tasks finished, then continue to step 3, until all tasks is done.
 // The above operations are completed in a transaction.
 // Finally, update the concurrent processing of the total number of rows, and store the completed handle value.
-func (w *worker) addPhysicalTableIndex(t table.PhysicalTable, indexInfo *model.IndexInfo, reorgInfo *reorgInfo) error {
+func (w *worker) writePhysicalTableRecord(t table.PhysicalTable, bfWorkerType backfillWorkerType, indexInfo *model.IndexInfo, oldColInfo, colInfo *model.ColumnInfo, reorgInfo *reorgInfo) error {
```

Comment:
It's a bit weird that the function is in `index.go` after renaming, how about moving it to `backfilling.go` maybe?

Reply:
Yes, I will put the functions shared by "addIndex" and "updateColumn" to `backfilling.go`.